core/CronArchive.php

```diff
@@ -1383,169 +829,176 @@ private function getApiToInvalidateArchivedReport()
 
     public function invalidateArchivedReportsForSitesThatNeedToBeArchivedAgain()
     {
+        $this->logger->info("Checking for queued invalidations...");
```

Comment:
Explanation part 6: invalidation is done as follows:

- first we look through every site/date in the getRememberedArchivedReportsThatShouldBeInvalidated() list. These are invalidated even if there are no visits for the period.
- then for every website we're archiving for, we invalidate the today archive if we need to.
- then for every website we're archiving for, we invalidate the yesterday archive if we need to (only done if the ts_archived for yesterday is not set for today, which means last time it was archived was before midnight)
- then we invalidate the custom range archives for every site to archive.
- check for segments that have been changed/updated recently and invalidate past data if needed.

Invalidating means the archives will be pulled in the main loop and processed.